### PR TITLE
Adds support for creating wallet in a non interactive way

### PR DESCRIPTION
**Pull request by PeGa**

### CODE REVIEW

File: docs/mine-hnt/validators/wallet.mdx
Comment (lthiery) [inline docs/mine-hnt/validators/wallet.mdx]:
```suggestion

```

File: docs/mine-hnt/validators/wallet.mdx
Comment (lthiery) [inline docs/mine-hnt/validators/wallet.mdx]:
```suggestion
HELIUM_WALLET_PASSWORD="My Password" ./helium-wallet create basic --network testnet
```

Comment (lthiery) [reply docs/mine-hnt/validators/wallet.mdx]: I would recommend this one  liner instead

File: docs/mine-hnt/validators/wallet.mdx
Comment (lthiery) [inline docs/mine-hnt/validators/wallet.mdx]:
```suggestion
#### Create or Use Wallet without Password Input
```

File: docs/mine-hnt/validators/wallet.mdx
Comment (lthiery) [inline docs/mine-hnt/validators/wallet.mdx]:
```suggestion
For security reasons, it is generally recommended that you type in your password every time. Storing the password in an environmental variable makes it very easy for the password to get compromised. Sometimes though, despite the security risks, you may want to interact with the CLI without need to input the password. In this case, using the environmental variable `HELIUM_WALLET_PASSWORD` enables this.
```

**Review by lthiery — CHANGES_REQUESTED** [review]
I wanted to generalize these instructions a little bit and provide some warning/context.

In addition, I prefer the oneliner, but take it or leave it.

File: docs/mine-hnt/validators/wallet.mdx
Comment (PeGa) [reply docs/mine-hnt/validators/wallet.mdx]: That works too, although in this one-line version exporting the variable doesn't make a lot of sense (besides wanted a little more of visual clarity), but works for me!

**Comment by PharkMillups** [comment]
Thanks, @PeGa! Can you go through and do a quick review of all these suggestions from @lthiery? They look good to me but you should review before we merge.

File: docs/mine-hnt/validators/wallet.mdx
Comment (PeGa) [reply docs/mine-hnt/validators/wallet.mdx]: Precisely like this. The importance of not exporting HELIUM_WALLET_PASSWORD lies in the fact once the process has finished, the password will remain in memory. However, in the change proposed here (without the `export` command, the variable will be only availble to the helium-wallet process and nothing else.

File: docs/mine-hnt/validators/wallet.mdx
Comment (PeGa) [reply docs/mine-hnt/validators/wallet.mdx]: The clarification is valid and explains why the variable exists, however I'd like to insist in specifying an example (like [this one](https://github.com/helium/docs/pull/218/files/4791a853daf3bccc8adc467a4ce93079a75e8a8f#diff-29853b474d408aac49fc4717bb1e6b78e0c1efc2d5b5b7c54c106a1926c962b1) so nobody is encouraged to use `export`.

**Comment by PeGa** [comment]
Thank you for the heads up @PharkMillups , answered all relevant items.

File: docs/mine-hnt/validators/wallet.mdx
Comment (lthiery) [reply docs/mine-hnt/validators/wallet.mdx]: I think we're in agreement. Removing the `export` line and injecting it during the executable invocation is what I've proposed  here. I'm waiting for you to accept the suggestions since it's your PR.

File: docs/mine-hnt/validators/wallet.mdx
Comment (lthiery) [reply docs/mine-hnt/validators/wallet.mdx]: I am suggesting removal of this line.

File: docs/mine-hnt/validators/wallet.mdx
Comment (lthiery) [reply docs/mine-hnt/validators/wallet.mdx]: This is the proposal. As I mentioned earlier, please accept the suggestions if you agree. I don't want to push to your branch if I can avoid it.

Comment (PeGa) [reply docs/mine-hnt/validators/wallet.mdx]: @lthiery Agree to the modifications!

File: docs/mine-hnt/validators/wallet.mdx
Comment (lthiery) [inline docs/mine-hnt/validators/wallet.mdx]:
```suggestion
For security reasons, it is generally recommended that you type in your password every time. Storing the password in an environmental variable makes it very easy for the password to get compromised. Sometimes though, despite the security risks, you may want to interact with the CLI without inputting the password. In this case, using the environmental variable `HELIUM_WALLET_PASSWORD` enables this.
```

File: docs/mine-hnt/validators/wallet.mdx
Comment (lthiery) [inline docs/mine-hnt/validators/wallet.mdx]:
```suggestion
HELIUM_WALLET_PASSWORD=p4ssw0rd ./helium-wallet create basic --network testnet
```

**Review by lthiery — APPROVED** [review]

**Pull request merged**
